imio/dms/mail/skins/imio_dms_mail/object_rename_title.py

```
@@ -0,0 +1,32 @@
+REQUEST = context.REQUEST
+title = safe_unicode(context.title_or_id())
+
+mtool = getToolByName(context, 'portal_membership')
+if not mtool.checkPermission('Modify portal content', context):
+    raise Unauthorized(_(u'Permission denied to rename ${title}.',
+                         mapping={u'title': title}))
+
+pathName = url_quote_plus('paths:list')
+safePath = '/'.join(context.getPhysicalPath())
+orig_template = REQUEST['HTTP_REFERER'].split('?')[0]
+url = '%s/folder_rename_title_form?orig_template=%s&%s=%s' % (context.absolute_url(),
```

Ruff (0.15.12)

[error] Undefined name `context`

(F821)